cmd/ocm/create/cluster/cmd.go

```diff
@@ -514,7 +527,20 @@ func preRun(cmd *cobra.Command, argv []string) error {
 		parseSubscriptionType(args.subscriptionType) == parseSubscriptionType(gcpSubscriptionTypeTemplate.Value)
 
 	if isGcpMarketplace {
-		fmt.Println("setting provider to", c.ProviderGCP)
+		if args.interactive {
+			fmt.Println("setting provider to", c.ProviderGCP)
+			fmt.Println("setting ccs to 'true'")
+			args.ccs.Enabled = true
+			fmt.Println("Review and accept Google Terms and Agreements on", gcpTermsAgreementsHyperlink)
+			err = arguments.PromptBool(fs, "marketplace-gcp-terms")
+			if err != nil {
+				return err
+			}
+		}
+		if !args.marketplaceGcpTerms {
+			return fmt.Errorf(gcpTermsAgreementError)
+		}
+		args.ccs.Enabled = true
 		args.provider = c.ProviderGCP
```

Comment:
I would have these assignments close  to the code where we print messages related to the override

Reply:
Added them after the prompt messages